watson_developer_cloud/discovery_v1.py

```diff
@@ -226,6 +226,29 @@ def create_collection(self,
                             params={'version': self.version},
                             accept_json=True)
 
+    def update_collection(self,
+                          environment_id,
+                          collection_id,
+                          name,
+                          description="",
+                          configuration_id=None):
+
+        if configuration_id is None:
```

Comment:
If so we can just remove these lines, and having it set to `None` will cause it to be removed from the request which should allow the server to do the right thing automatically.